docs/tutorials/python/tutorial_scripts/file.py

```diff
@@ -0,0 +1,109 @@
+"""
+Here is where you'll find the code for the File tutorial.
+"""
+# Step 1: Upload several files to Synapse
+import os
+import synapseclient
+import synapseutils
+from synapseclient import File
+
+syn = synapseclient.login()
+
+# Retrieve the project ID
+my_project_id = syn.findEntityId(
+    name="My uniquely named project about Alzheimer's Disease"
+)
+
+# Retrieve the IDs of the folders I want to upload to
+batch_1_folder = syn.findEntityId(
```

Comment:
This is a nice use of `findEntityId`

I don't know if we want to outline that you can use the UI to find the synapse id

Reply:
For this I am going to focus solely on using this programmatic approach. I am only planning to mention the UI to look at results of doing something.